components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1279,7 +1279,7 @@ void RewardsServiceImpl::LoadURL(
   auto request = std::make_unique<network::ResourceRequest>();
   request->url = GURL(url);
   request->method = request_method;
-  request->allow_credentials = false;
+  request->allow_credentials = true;
```

Comment:
is this only needed for Twitter? can this be put in an `if` block so it's only `true` for Twitter requests?

Reply:
Yes, I believe it's only needed by Twitter at the moment (as far as the media sites we currently support). I can definitely add a guard to make this Twitter-only.

Reply:
Fixed and pushed.